decidim/decidim
# Fix 500 error when creating the Conference partner with an invalid image

#### :tophat: What? Why?
Updating a Conference Partner with a picture with more that 3840px, will raise an ActiveRecord::RecordInvalid, as the the conference partner is created using Decidim.traceability.create! .

#### :pushpin: Related Issues
- Related to #7188
- Related to #7189
- Related to #7210

#### Testing
1. Go to Conference partner admin section
2. Add a partner
3. Go to the image upload
4. Attach the map image from this ticket
5. See the error
6. Apply the patch
7. See the error will become a form error

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*

![800px-Pizigani_1367_Chart_10MB-1](https://user-images.githubusercontent.com/105683/105573069-a701f180-5d63-11eb-9818-197739ce5381.jpeg)

:hearts: Thank you!
